addons/openebs/2.6.0/install.sh

```diff
@@ -253,13 +259,247 @@ function openebs_upgrade() {
+    logSubstep "OpenEBS batch job to upgrade cStor pools completed."
+}
+
+
+function openebs_upgrade_cstor() {
+    # upgrade job from https://github.com/openebs/upgrade/blob/v2.6.0/examples/upgrade/cstor-volume.yaml
+
+    if [[ -z $PREVIOUS_OPENEBS_VERSION ]] || [[ "$PREVIOUS_OPENEBS_VERSION" -eq "2.6.0" ]]; then
+      return 0 # no upgrade needed
+    fi
+
+    local pvs
+    pvs=$(kubectl get pv --template '{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}')
+
+    echo "" > /tmp/openebs-volume-upgrade.yaml.part
+    for value in $pools; do
+      echo "        - \"$value\"" >> /tmp/openebs-volume-upgrade.yaml.part
+    done
+
+    logSubstep "Upgrading cstor volumes $pvs from $PREVIOUS_OPENEBS_VERSION to $OPENEBS_VERSION"
+    local out_file=/tmp/openebs-volume-upgrade.yaml
+    cat <<UPGRADE_VOLUME >$out_file
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: cstor-volume-upgrade
+  namespace: $OPENEBS_NAMESPACE
+spec:
+  backoffLimit: 4
+  template:
+    spec:
+      serviceAccountName: openebs-maya-operator
+      containers:
+      - name:  upgrade
+        args:
+        - "cstor-volume"
+
+        # --from-version is the current version of the volume
+        - "--from-version=$PREVIOUS_OPENEBS_VERSION"
+
+        # --to-version is the version desired upgrade version
+        - "--to-version=2.6.0"
+        # if required the image prefix of the volume deployments can be
+        # changed using the flag below, defaults to whatever was present on old
+        # deployments.
+        #- "--to-version-image-prefix=openebs/"
+        # if required the image tags for volume deployments can be changed
+        # to a custom image tag using the flag below,
+        # defaults to the --to-version mentioned above.
+        #- "--to-version-image-tag=ci"
+
+        # VERIFY that you have provided the correct list of volume Names
+$(cat /tmp/openebs-volume-upgrade.yaml.part)
+
+        # Following are optional parameters
+        # Log Level
+        - "--v=4"
+        # DO NOT CHANGE BELOW PARAMETERS
+        env:
+        - name: OPENEBS_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        tty: true
+
+        # the image version should be same as the --to-version mentioned above
+        # in the args of the job
+        image: openebs/upgrade:2.6.0
+        imagePullPolicy: IfNotPresent
+      restartPolicy: OnFailure
+UPGRADE_VOLUME
+
+    echo "cstor-volume-upgrade file:"
+
+    kubectl apply -f $out_file
+    logSubstep "Waiting for cstor-volume-upgrade job"
+    spinner_until 240 job_is_completed "$OPENEBS_NAMESPACE" "cstor-volume-upgrade"
+    logSubstep "OpenEBS batch job to upgrade cstor volumes completed."
+}
+
+# returns true when a job has completed
+# TODO: remove before merging
+function job_is_completed() {
+  local namespace="$1"
+  local jobName="$2"
+  kubectl get jobs -n "$namespace" "$jobName" | grep -q '1/1'
+}
```

Comment:
Guessing you can delete this now the `common.sh` is the permanent home.

Reply:
not if I want it to keep working in testgrid